beacon-chain/sync/initial-sync/blocks_queue.go

```diff
@@ -189,6 +189,13 @@ func (q *blocksQueue) loop() {
 			q.cancel()
 		}
 
+		log.WithFields(logrus.Fields{
+			"highestExpectedSlot":       q.highestExpectedSlot,
+			"noRequiredPeersErrRetries": q.exitConditions.noRequiredPeersErrRetries,
+			"headSlot":                  q.headFetcher.HeadSlot(),
+			"state":                     q.smm,
+		}).Debug("tick")
```

Comment:
is this too verbose? would trace be better?

Reply:
- Nope, I think, trace is so noisy (all the net packages), that tick is also quite frequent (still manageable, when one is debugging). 
- Since it is a debug (and I suppose/hope a temporary one), I believe it is important to have those logs. I use them when debugging, otherwise it is hard to figure out what's going on in FSMs.

Reply:
sounds good 👍